gpiozero/boards.py

```diff
@@ -178,9 +200,96 @@ def _blink_device(self, on_time, off_time, fade_in_time, fade_out_time, n, fps=5
                 break
 
 
+class LEDBarGraph(LEDCollection):
+    """
+    Extends :class:`CompositeDevice` to control a line of LEDs representing a
+    bar graph. Positive values (0 to 1) light the LEDs from first to last.
+    Negative values (-1 to 0) light the LEDs from last to first.
+
+    The following example turns on all the LEDs on a board containing 5 LEDs
+    attached to GPIO pins 2 through 6::
```

Comment:
Umm, looks like the example only turns on the first two LEDs and the last two LEDs, but not **all** LEDs?